packages/build/tests/monitor/snapshots/tests.js.md

```diff
@@ -1822,43 +1972,56 @@ Generated by [AVA](https://avajs.dev).
       Bundling of function "trouble" failed                         ␊
     ────────────────────────────────────────────────────────────────␊
     ␊
       Error message␊
-      Build failed with 1 error:␊
-      packages/build/tests/monitor/fixtures/function_bundling_error/netlify/functions/trouble.ts ERROR: Cannot assign to "bar" because it is a constant␊
+      Command failed with exit code 101: cargo build --target x86_64-unknown-linux-musl --release␊
+        Updating crates.io index␊
+      Downloading crates ...␊
+      Downloaded tokio v1.0.0␊
+      Downloaded tower v1.0.0␊
+      Downloaded serde v1.0.0␊
+       Compiling tokio v1.0.0␊
+       Compiling tower v1.0.0␊
+       Compiling serde v1.0.0 (/external/path)␊
+      error: expected one of \`!\` or \`::\`, found keyword \`use\`␊
+    STACK TRACE␊
+      |␊
+      1 | KB␊
+      |   - expected one of \`!\` or \`::\`␊
+      2 | use aws_lambda_events::{␊
+      | ^^^ unexpected token␊
     ␊
       Error location␊
       While bundling function "trouble"␊
     ␊
       Resolved config␊
       build:␊
-        publish: packages/build/tests/monitor/fixtures/function_bundling_error␊
+        publish: packages/build/tests/monitor/fixtures/serverless_function␊
         publishOrigin: default␊
-      functionsDirectory: packages/build/tests/monitor/fixtures/function_bundling_error/netlify/functions␊
+      functionsDirectory: packages/build/tests/monitor/fixtures/serverless_function/netlify/functions␊
     ␊
     Error monitoring payload:␊
     {␊
       "errorClass": "functionsBundling",␊
-      "errorMessage": "Build failed with 1 error:/npackages/build/tests/monitor/fixtures/function_bundling_error/netlify/functions/trouble.ts ERROR: Cannot assign to /"bar/" because it is a constant",␊
+      "errorMessage": "Command failed with exit code 101: cargo build --target x86_64-unknown-linux-musl --release/n  Updating crates.io index/nDownloading crates .../nDownloaded tokio v1.0.0/nDownloaded tower v1.0.0/nDownloaded serde v1.0.0/n Compiling tokio v1.0.0/n Compiling tower v1.0.0/n Compiling serde v1.0.0 (/external/path)/nerror: expected one of \`!\` or \`::\`, found keyword \`use\`/n--> src/main.rs:2:1/n|/n1 | KB/n|   - expected one of \`!\` or \`::\`/n2 | use aws_lambda_events::{/n| ^^^ unexpected token",␊
       "context": "'Bundling of serverless function failed",␊
-      "groupingHash": "'Bundling of serverless function failed/nBuild failed with 0 error:/n/file/path ERROR: Cannot assign to /"bar/" because it is a constant",␊
+      "groupingHash": "'Bundling of serverless function failed/nCommand failed with exit code 0: cargo build --target x0_0-unknown-linux-musl --release/n  Updating crates.io index/nDownloading crates .../nDownloaded cratesCompiled crates/file/path/nerror: expected one of \`\` or \`\`, found keyword \`\`/n--> /external/path | KB/n|   - expected one of \`\` or \`\`/n0 | use aws_lambda_events::{/n| ^^^ unexpected token",␊
```

Comment:
Why is there a single quote at the beginning?

I wonder if in this case it would make sense to simply group by 
```
'Bundling of serverless function failed/nCommand failed with exit code 0: cargo build
```

Reply:
> Why is there a single quote at the beginning?

You ask difficult questions! I'm not sure. 🤔 



> I wonder if in this case it would make sense to simply group by ?

That would mean normalising away all parameters in commands, which feels a bit too extreme? I would rather start with small steps that work well for most cases and then add to them, than go too far and then having to take stuff back.

Reply:
I just wondered if the actual error message and code excerpt make sense to include.